lib/video_info.rb

```diff
@@ -41,11 +41,29 @@ def ==(other)
     url == other.url && video_id == other.video_id
   end
 
+  @@disable_providers = []
+
+  def self.disable_providers
+    @@disable_providers
+  end
+
+  def self.disable_providers=(providers)
+    @@disable_providers = providers
+  end
+
   private
 
   def _select_provider(url, options)
     if provider_const = _providers_const.detect { |p| p.usable?(url) }
-      provider_const.new(url, options)
+      const_provider = provider_const.new(url, options)
+      
+      if defined? const_provider.provider and const_provider.provider
+        if self.class.disable_providers.map { |p| p.downcase }.include? const_provider.provider.downcase
```

Comment:
Line is too long. [104/80]<br>Pass `&:downcase` as an argument to `map` instead of a block.